content/it/_index.html

```diff
@@ -1,62 +1,70 @@
 
-Progettato secondo gli stessi principi che consentono a Google di gestire miliardi di container alla settimana, Kubernetes può scalare senza aumentare il tuo team operativo.
+Nota: non tutte le distro vengono mantenute attivamente. Scegli le distro che sono state testate con una versione recente di Kubernetes.
 
-{{% /blocks/feature %}}
+## Managing a cluster
 
-{{% blocks/feature image="blocks" %}}
-#### Never Outgrow
+* [Gestione di un cluster](/docs/tasks/administration-cluster/cluster-management/) descrive diversi argomenti relativi al ciclo di vita di un cluster: creazione di un nuovo cluster, aggiornamento dei nodi master e worker del cluster, esecuzione della manutenzione del nodo (ad esempio kernel aggiornamenti) e aggiornamento della versione dell'API di Kubernetes di un cluster in esecuzione.
 
-Che si tratti di eseguire test localmente o di gestire un'azienda globale, la flessibilità di Kubernetes cresce con te per fornire le tue applicazioni in modo coerente e semplice, indipendentemente dalla complessità delle tue esigenze.
-{{% /blocks/feature %}}
+* Scopri come [gestire i nodi](/docs/concepts/nodi/node/).
 
-{{% blocks/feature image="suitcase" %}}
-#### Run Anywhere
+* Scopri come impostare e gestire la [quota di risorse](/docs/concepts/policy/resource-quote/) per i cluster condivisi.
 
-Kubernetes è open source e ti offre la libertà di trarre vantaggio dall'infrastruttura cloud locale, ibrida o pubblica, consentendo di spostare facilmente i carichi di lavoro nel punto in cui è importante per te.
-{{% /blocks/feature %}}
+## Proteggere un cluster
 
-{{< /blocks/section >}}
+* [Certificati](/docs/concepts/cluster-administration/certificates/) descrive i passaggi per generare certificati utilizzando diverse catene di strumenti.
 
-{{< blocks/section id="video" background-image="kub_video_banner_homepage" >}}
-<div class="light-text">
-    <h2>Le sfide della migrazione di oltre 150 microservizi a Kubernetes</h2>
-    <p>By Sarah Wells, Technical Director for Operations and Reliability, Financial Times</p>
-    <button id="desktopShowVideoButton" onclick="kub.showVideo()">Guarda i Video</button>
-    <br>
-    <br>
-    <br>
-    <a href="https://events.linuxfoundation.org/events/kubecon-cloudnativecon-north-america-2019" button id="desktopKCButton">Attend KubeCon in San Diego on Nov. 18-21, 2019</a>
-    <br>
-    <br>
-    <br>
-    <br>
-    <a href="https://events.linuxfoundation.org/events/kubecon-cloudnativecon-europe-2020/" button id="desktopKCButton">Attend KubeCon in Amsterdam on Mar. 30-Apr. 2, 2020</a>
-</div>
-<div id="videoPlayer">
-    <iframe data-url="https://www.youtube.com/embed/H06qrNmGqyE?autoplay=1" frameborder="0" allowfullscreen></iframe>
-    <button id="closeButton"></button>
-</div>
-{{< /blocks/section >}}
+* [Kubernetes Container Environment](/docs/concepts/containers/container-environment-variables/) descrive l'ambiente per i contenitori gestiti Kubelet su un nodo Kubernetes.
```

Comment:
`contenitori` -> `container`

Are you still using google translate?

Reply:
Nope, maybe I forgot from the previous version. sorry about that